        -:    0:Source:c:/users/user/desktop/lab4/appexception.h
        -:    0:Graph:C:\Users\User\Desktop\lab4\cmake-build-debug-coverage\CMakeFiles\lab4.dir\Vector\DynamicArray.cpp.gcno
        -:    0:Data:C:\Users\User\Desktop\lab4\cmake-build-debug-coverage\CMakeFiles\lab4.dir\Vector\DynamicArray.cpp.gcda
        -:    0:Runs:1
        -:    0:Programs:1
        -:    1://
        -:    2:// Created by User on 3/19/2021.
        -:    3://
        -:    4:
        -:    5:#ifndef MAIN_CPP_APPEXCEPTION_H
        -:    6:#define MAIN_CPP_APPEXCEPTION_H
        -:    7:
        -:    8:#endif //MAIN_CPP_APPEXCEPTION_H
        -:    9:
        -:   10:#pragma once
        -:   11:#include <exception>
        -:   12:using namespace std;
    #####:   13:class AppException :public exception {
    $$$$$:   13-block  0
    $$$$$:   13-block  1
    $$$$$:   13-block  2
    $$$$$:   13-block  3
    $$$$$:   13-block  4
    $$$$$:   13-block  5
        -:   14:    using exception::exception;
        -:   15:};
